Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -434,7 +434,7 @@ private void SetNextStatusUpdate()
                         AlphaRuntimeStatistics = AlphaRuntimeStatistics
                     };
 
-                    SaveResults($"live-{JobId}.json", result);
+                    SaveResults($"L-{JobId}.json", result);
```

Comment:
Partly out of curiosity and also documentation for posterity -- is there any code that depends on this file name? If so, we may want to document that somewhere (lean may not be the best place for it depending on where the dependencies are).

Reply:
This `.json` file was added yesterday in PR https://github.com/QuantConnect/Lean/pull/3540 so no real consumer yet 👍.